addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CrossDomainScriptInclusionScanRuleUnitTest.java

```diff
@@ -362,4 +375,41 @@ void crossDomainScriptInContextLow() throws HttpMalformedHeaderException {
                 alertsRaised.get(0).getEvidence(),
                 equalTo("<script src=\"https://www.otherDomain.com/script2\"/>"));
     }
+
+    @Test
+    void dontRaiseIssueWhenOneLinkWithBlankTargetTrustedDomain()
+            throws HttpMalformedHeaderException {
+        // Given
+        HttpMessage msg = new HttpMessage();
+        msg.setRequestHeader("GET https://www.example.com/test/ HTTP/1.1");
+        rule.getConfig()
+                .setProperty(RuleConfigParam.RULE_DOMAINS_TRUSTED, "https://www.example2.com/.*");
+        // When
+        msg.setResponseBody(
+                "<html><a href=\"https://www.example2.com/page1\" rel=\"opener\" target=\"_blank\">link</a></html>");
```

Comment:
This needs to be updated to match this rule (e.g. add `script` elements with `src` attribute, review above methods).

Reply:
Updated

Reply:
Test names should also be updated to match what's being tested.

Reply:
Updated names of test methods.